cms/templates/js/course-outline.underscore

```diff
@@ -70,7 +70,7 @@ if (is_proctored_exam) {
 <li class="outline-item outline-<%- xblockType %> <%- visibilityClass %> is-draggable <%- includesChildren ? 'is-collapsible' : '' %> <%- isCollapsed ? 'is-collapsed' : '' %>"
     data-parent="<%- parentInfo.get('id') %>" data-locator="<%- xblockInfo.get('id') %>">
 
-    <span class="draggable-drop-indicator draggable-drop-indicator-before"><i class="icon fa fa-caret-right"></i></span>
+    <span class="draggable-drop-indicator draggable-drop-indicator-before"><span class="icon fa fa-caret-right" aria-hidden="true"></span></span>
```

Comment:
@cahrens, @clrux said we should add sr-text to the collapse/expand icons. If you go to the course outline page (in studio) though, when you hover over the icon, there's text that says "Collapse/expand this section". Hovering over the header text also gives that text. Can a screenreader pick up on that, and if it can, do you think it would be confusing to have two places that are both supposedly links to expand/collapse?